notebooks/en/advanced_rag.ipynb

```diff
@@ -6,7 +6,7 @@
     "id": "hUCaGdAj9-9F"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    # Plot the distrubution of document lengths, counted as the number of tokens.
</pre><p>Typo here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
I have corrected this typo too.